controllers/validator.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+
+function validateObject(obj, schema) {
+
+    let errorUnexpectedMessage;
+    let errorMessage;
+
+    let unexpected = _.difference(Object.keys(obj), Object.keys(schema));
```

Comment:
So much unexpectedness 🐤 
Could you please name these variables more specificly?